osbuild/api.py

```diff
@@ -193,6 +201,18 @@ def _cleanup(self):
             os.close(self._output_pipe)
             self._output_pipe = None
 
+def exception(path="/run/osbuild/api/osbuild"):
```

Comment:
How about passing the exception as an argument, rather than relying on ambient/global data?